experiments/getNumberOrdinal.js

```diff
@@ -0,0 +1,17 @@
+//Returns the correct ordinal suffix for a number
+//ex. 1st, 2nd, 3rd etc. 
+skuid.formula.Formula (
+  'GET_ORDINAL',
```

Comment:
This function is useful, but is it relevant to your experiment? I don't see that it is used anywhere. Unless I'm missing something...

Reply:
@elenahatch Gonna hazard a guess that you branched from your other PR's branch (#23 ) ? If so, could you remove this file from this PR?

Reply:
Oh, oops. No - it's not related to this.

Reply:
@itmightbecody yes. Will do.